packages/config/config/devices/0x013c/pat02-a.json

```diff
@@ -0,0 +1,338 @@
+// Philio Technology Corp PAT02-A
+// Flood Sensor
+{
+	"manufacturer": "Philio Technology Corp",
+	"manufacturerId": "0x013c",
+	"label": "PAT02-A",
+	"description": "Flood Sensor",
+	"devices": [
+		{
+			"productType": "0x0002",
+			"productId": "0x001f"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"1": {
+			"label": "Basic Set Dim Level (Off)",
+			"description": "When the flood trigger is off, send this dim level via the Basic CC to Group 2.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Basic Set Dim Level (On)",
+			"description": "When the flood trigger is on, send this dim level via the Basic CC to Group 2.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"5[0x00]": {
```

Comment:
Bit 0? That needs to be 0x01 then

Reply:
Documentation says value 1 deactivates flood sensor

Reply:
Too little sleep. I'll fix these.